fr.esrf.tango.generator.xtend/src/fr/esrf/tango/pogo/generator/python/PythonDeviceHL.xtend

```diff
@@ -212,13 +214,31 @@ from tango.server import Device, DeviceMeta
 «IF !cls.classProperties.empty || !cls.deviceProperties.empty»from tango.server import «IF !cls.classProperties.empty»class_property, «ENDIF»«IF !cls.deviceProperties.empty»device_property«ENDIF»«ENDIF»
 from tango import AttrQuality, DispLevel, DevState
 from tango import AttrWriteType, PipeWriteType
+from enum import IntEnum

```

Comment:
We only need the import if there is at least one enum.  Not sure if it is easy to determine that?

Reply:
Agree. Updated the condition before importing the IntEnum.